blocks/init/src/Blocks/custom/column/components/column-options.js

```diff
@@ -138,6 +145,32 @@ export const ColumnOptions = ({ attributes, setAttributes }) => {
 				})}
 			</Responsive>
 
+			<Responsive
+				label={
+					<Fragment>
+						<Icon icon={icons.spacingTop} />
+						{__('Align', 'eightshift-frontend-libs')}
+					</Fragment>
+				}
+			>
+				{align.map((item, index) => {
+
+					const point = ucfirst(options.breakpoints[index]);
+					const attr = `align${point}`;
+
+					return (
+						<Fragment key={index}>
+							<SelectControl
```

Comment:
Would a `ButtonGroup` with `IconButton`s inside have better UX than a `SelectControl`?

e.g.:
![image](https://user-images.githubusercontent.com/77000136/105961441-cad47880-607e-11eb-8adc-31e29c550c9c.png)

Reply:
Sure it would. Make an PR for this change

Reply:
i like it